src/main/java/org/opentripplanner/routing/core/State.java

```diff
@@ -114,8 +123,8 @@ public State(Vertex vertex, Edge backEdge, long timeSeconds, long startTime, Rou
         // this should be harmless since reversed clones are only used when routing has finished
         this.stateData.opt = options;
         this.stateData.startTime = startTime;
+        this.time = timeSeconds * 1000;
```

Comment:
It was moved up because additional time can be added to the start time if beginning a trip in a TNC vehicle that has an ETA associated with it. 

I'll add a comment.